### PR TITLE
New arabic language package

### PR DESCRIPTION
**Pull request by msaied93**
Hey, I added Arabic translation; however, there is still more work to be done: the page direction should become RTL when choosing Arabic language... not sure how to do it.

### CODE REVIEW

**Comment by cyntss** [comment]
@msaied93 thank you for the contribution. Can you try to look into how to change the direction to RTL ?
I will try to help you when I have some time, but at the moment im solving many other issues we have so I would really appreciate it if you could try to fix it yourself :smiley:

**Comment by msaied93** [comment]
Well, I guess that does the trick.

File: assets/js/opensuse-theme.js [hunk 1]
```diff
@@ -252,6 +253,19 @@ $(document).on("ready", function(){
   }
 
 });
+
+//change page direction when changing lang into/from ar
+$(window.lang).on("afterUpdate", function (event, currentLang, newLang, pack1, pack2) {
```
Comment (cyntss) [inline assets/js/opensuse-theme.js]: why are you passing the arguments event, pack1 and pack2 ? I dont see where you are using them below

**Comment by cyntss** [comment]
For some reason the News are no longer visible when we change to RTL

**Comment by cyntss** [comment]
There seem to be other conflicts with the animation and general layout. This PR will require much more work than just changing the css to direction: rtl. IMO.

**Comment by coolo** [comment]
Can you move the actual translations to subversions?

**Comment by lnussel** [comment]
I've opened #81 to implement the missing features, closing

**Pull request closed without merging**